### ISSUE TITLE
[Suggestion] Tool Textures

### ISSUE THREAD

**Issue by VallenFrostweaver** [RootsTeam/Animunda#6]
I love the purple theme to this mod as it really lends itself well (though it looks weird in still images for some reason - I guess it's just dark). Upon crafting the Void Lense (should be spelled Lens BTW) I saw that transparent texture on it and it looks great! I was thinking that you should apply this transparency to the Void tools/weapons etc. as well for a really cool effect for the entire mod.

**Comment by ShadowGamerXY** [comment]
Yeah I really like the idea, I will go and tinker with it and see what I can come up with. And I know the Lense  was a typo, sorry my mistake.

**Comment by ShadowGamerXY** [comment]
So the way that I am going to implement this is connected to the infusing of tools. So basically the more you infuse the tool/armor the more powerful it gets and the less connection it has to the material world... meaning that its "texture" will get more and more transparent... This is the base idea.

**Comment by VallenFrostweaver** [comment]
This sounds promising. :)
